src/metabase/types.clj

```diff
@@ -43,6 +55,8 @@
 (derive :type/SerializedJSON :type/Text)
 (derive :type/SerializedJSON :type/Collection)
 
+(derive :type/PostgresEnum :type/Text)
```

Comment:
Is this a PostgresEnum because the other databases that support enums are different enough that they will require their own separate/specific type?

Reply:
[Postgres enums are strings for querying purposes](https://www.postgresql.org/docs/current/static/datatype-enum.html) (e.g. you say `'toucan'` and not `0`) so for PostgresEnum it made sense to derive from Text. I don't know what other databases do but since enums are numbers in languages like `C` it didn't feel safe to me to assume they'd behave the same way.

Reply:
If it turns out other databases effectively treat them as text as well we can always put a common ancestor in between PostgresEnum and Text but for the time being I didn't want to make any incorrect assumptions.

On second thought I suppose `PostgresEnum` could actually be renamed something like `TextEnum`